core/shared/platform/linux/platform_internal.h

```diff
@@ -126,6 +130,10 @@ typedef int os_file_handle;
 typedef DIR *os_dir_stream;
 typedef int os_raw_file_handle;
 
+typedef struct pollfd os_poll_file_handle;
+typedef nfds_t os_nfds_t;
+typedef struct timespec os_timespec;
```

Comment:
duplicated definitions, maybe we can remove L67 to L70?

Reply:
removed L67 to L70